src/CoreWCF.Primitives/src/CoreWCF/Security/SamlSerializer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Xml;
+
+namespace CoreWCF.Security
+{
+    public class SamlSerializer
+    {
+        internal void PopulateDictionary(IXmlDictionary xmlDictionary)
```

Comment:
I can't find a reference to this method in the code anywhere

Reply:
right... removed the method just kept the signature class for future use..